lib/GeoIP2/Database/Reader.pm

```diff
@@ -103,7 +104,7 @@ sub city {
     my $self = shift;
     return $self->_model_for_address(
         'City',
-        type_check => qr/^(?:GeoLite2|GeoIP2)-(?:Precision-)?City$/,
+        type_check => qr/^(?:GeoLite2|GeoIP2)-(?:Precision-|)?City(-[a-zA-Z]+)?$/,
```

Comment:
Fixed, but I've now decided never to validate anything again. It's just too much work!